SYCL/Basic/fpga_tests/fpga_aocx.cpp

```diff
@@ -11,15 +11,17 @@
 /// E2E test for AOCX creation/use/run for FPGA
 // Produce an archive with device (AOCX) image. To avoid appending objects to
 // leftover archives, remove one if exists.
+// FIXME Disabled use of devicelib by assert feature until the 2-step build gets
+// fixed.
```

Comment:
Will it be fixed by https://github.com/intel/llvm/pull/3767 ?

Reply:
This one won't be fixed by intel/llvm#3767. There's an issue in driver which isn't connected to intel/llvm#3767. The issue can be encountered when using any devicelib function and compiling the same way it's done in this test. @mdtoguchi  is aware of the issue.